show/main.py

```diff
@@ -968,11 +961,13 @@ def version(verbose):
     click.echo("Build commit: {}".format(version_info['commit_id']))
     click.echo("Build date: {}".format(version_info['build_date']))
     click.echo("Built by: {}".format(version_info['built_by']))
-    click.echo("\nPlatform: {}".format(platform))
-    click.echo("HwSKU: {}".format(hwsku))
-    click.echo("ASIC: {}".format(asic_type))
-    click.echo("ASIC Count: {}".format(asic_count))
-    click.echo("Serial Number: {}".format(serial_number.stdout.read().strip()))
+    click.echo("\nPlatform: {}".format(hw_info['platform']))
+    click.echo("HwSKU: {}".format(hw_info['hwsku']))
+    click.echo("ASIC: {}".format(hw_info['asic_type']))
+    click.echo("ASIC Count: {}".format(hw_info['asic_count']))
+    click.echo("Serial Number: {}".format(hw_info['serial']))
+    click.echo("Model Number: {}".format(hw_info['model']))
+    click.echo("Hardware Rev: {}".format(hw_info['revision']))
```

Comment:
Suggest to use Revision and not Rev (like we don't use Num instead of Number)

Reply:
My reasoning was trying to keep it relatively lined up visually with the rest of the values. I am happy to make the change though.

Reply:
like this:  serial_number_cmd = "sudo decode-syseeprom -s"    when DB is not avalaible, decode-syseeprom will decode the eeprom directly.